internal/ad/krb5.go

```diff
@@ -54,7 +57,7 @@ func TicketPath() (string, error) {
 	krb5ccPath := strings.TrimPrefix(krb5cc, "FILE:")
 	fileInfo, err := os.Stat(krb5ccPath)
 	if err != nil {
-		return "", fmt.Errorf(gotext.Get("%q does not exist or is not accessible: %v", krb5ccPath, err))
+		return "", errors.Join(ErrTicketNotPresent, err)
```

Comment:
As this is now part of the API, shouldn’t we change the test to asserts on the error type?